### ISSUE TITLE
Using deprecated library

### ISSUE THREAD

**Issue by Ultimation** [nwops/puppet-retrospec#99]
```
➜  ~ git:(branch name) ✗ gem install puppet-retrospec
Fetching: trollop-2.9.9.gem (100%)
!    The 'trollop' gem has been deprecated and has been replaced by 'optimist'.
!    See: https://rubygems.org/gems/optimist
!    And: https://github.com/ManageIQ/optimist
Successfully installed trollop-2.9.9
Fetching: retrospec-0.6.2.gem (100%)
```

**Comment by logicminds** [comment]
Thanks, shouldn't be a problem for now.  I'll update soon.

**Comment by logicminds** [comment]
Fixed in 1.7.0
